src/ViewItem.js

```diff
@@ -267,6 +275,20 @@ class ViewItem extends React.Component {
             <FormattedMessage id="ui-inventory.copyItem" />
           </Icon>
         </Button>
+        <Button
+          href={`/requests?itemBarcode=${firstItem.barcode}&layer=create`}
+          onClick={() => {
```

Comment:
Do we need the `onClick` handler? Looks like the `href` passes the same attributes to the same URL.

Reply:
That was my initial thought too but for some reason the `href` was not executing at all. I'm not sure why. Perhaps ActionMenu is catching and preventing the click events somehow.